Tests/NetworkServiceTests/NetworkErrorTests.swift

```diff
@@ -0,0 +1,29 @@
+import NetworkService
+import XCTest
+
+final class NetworkErrorTests: XCTestCase {
+    func testAllNetworkErrorsExistInAPI() {
+        // Given
+        let possibleNetworkErrors = Set(NetworkError.allCases)
+
+        // When
+        let expectedNetworkErrors = Set([
+            NetworkError.lostConnection,
```

Comment:
nit:
JFYI, set can be created like this as well. 
For me, it's more readable due to fewer parentheses.
```suggestion
        let expectedNetworkErrors: Set = [
            NetworkError.lostConnection,
```

Reply:
Fixed.